### DISCUSSION TITLE
Desaster recovery 2 masters switching off main-master

### DISCUSSION THREAD

**Discussion by moserwi** [moosefs/moosefs#533] [General]
I do face a problem.
the chunkservers are connected to master02. Both masters have metasloggers. It is not possible to bring up master01 which is started with option -a in moosefs-master.service

error: init: metadata manager failed !!!

Stopping master02... no way to bring up master01...
Once I had 2 masters running but the chunkservers do not connect to master01 when master 02 is off.

HowTo's? nope...

Grateful for any input.
Thank you

**Comment by chogata** [comment]
Hello,
I don't really understand your situation. You use 2 masters - do you have MooseFS pro version? If yes, then contact support@moosefs.com for help.

**Reply by moserwi** [reply]
No, community edition but nonetheless there should be a HowTo for manual desaster recovery in case of the master1 failure.
Just testing...

**Reply by chogata** [reply]
If you have a metalogger installed on a second machine and all your settings (paths, hostnames etc.) are default, it's as simple as stopping the metalogger process (yes, you have to stop it first) and starting the master process with -a option. If this doesn't work for you, then there has to be a configuration problem. I can help, but I need the full information form the master (all the lines it displays, not just the 'error: init: metadata manager failed !!!').

**Reply by moserwi** [reply]
OK Thank you so much. I will proceed testing by the end of next week and will report my efforts.

**Reply by moserwi** [reply]
meanwhile the setup:
5x rockpro64 4 GB RAM with 5x 2 TB m.2 NVME SSD
sbc1: master & metalogger & GUI  all active
sbc2: master & metalogger & GUI  all active, sbc2 is main master
sbc3 to sbc 5 chunkservers active

GUI an sbc2 reports everything clean and fine.

Next step is to stop main master (sbc2) and metalogger.
Stop metalogger on follower master
restart follower master with option -a
start metalogger again
---------------
Keep you informed.

**Comment by moserwi** [comment]
Hi,
well I am back at the server testbed again after 7 month of absence. I will continue testing within the next two months.
